Dockerfile

```diff
@@ -75,6 +75,8 @@ WORKDIR ${WORK_DIR}
 COPY --chown=user:user . ${SRC_DIR}
 
 # installs buildozer and dependencies
-RUN pip3 install --user Cython==0.28.6 ${SRC_DIR}
+RUN pip3 install --user wheel Cython==0.28.6 ${SRC_DIR}
+
+RUN python3 -m pip install -U pip virtualenv
```

Comment:
Thanks for the PR, and why the inconsistency one time with `pip3 install` and the other time `python3 -m pip install`?

Reply:
Just a reflex I have to use `python3 -m pip`
I change the line into 'pip3'